.circleci/config.yml

```diff
@@ -35,11 +29,13 @@ workflows:
       #       branches:
       #         only: master
 
-      # - hackage/upload:
-      #     context: haskell-ci
-      #     publish: true
-      #     requires:
-      #       - GitHub Release
-      #     username: ${HACKAGE_USER}
-      #     password: ${HACKAGE_PASS}
+      - hackage/upload:
+          context: haskell-ci
+          publish: true
+          filters:
+            branches:
+              only: master
+          # requires:
+          #   - GitHub Release
```

Comment:
Shouldn't we enable github releases too?

Reply:
I don't know how to do it now, so I kept it disabled.
I also have no time to work on GH orb right now.